npm/c/package.json

```diff
@@ -7,6 +7,6 @@
   "main": "index.js",
   "bin": { "bin_c": "bin.js" },
   "scripts": {
-    "postinstall": "echo \"{\\\"answer\\\":\\\"42*\\\"}\" > data.json"
+    "postinstall": "echo '{\"answer\":\"42*\"}' > data.json"
```

Comment:
guess you should bump the version to 2.0.1 and change any transitive references; I've been publishing to npm manually so I can do that after this lands